helm/charts/determined/templates/master-config.yaml

```diff
@@ -187,9 +187,8 @@ stringData:
       max_slots_per_pod: {{ required "A valid Values.maxSlotsPerPod entry is required!" .Values.maxSlotsPerPod }}
       master_service_name: determined-master-service-{{ .Release.Name }}
       {{- if .Values.defaultScheduler}}
-      {{- $schedulerType := .Values.defaultScheduler | trim}}
-      {{- if or (eq $schedulerType "coscheduler") (eq $schedulerType "preemption")}}
-      default_scheduler: {{ $schedulerType }}
+      {{- if eq (.Values.defaultScheduler | trim ) "coscheduler"}}
+      default_scheduler: "coscheduler"
```

Comment:
Should we be checking for "preemption" and setting it to the default?

Reply:
No need, the default is ""